src/App.tsx

```diff
@@ -15,36 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  default = 'default',
+  alphabetically = 'alphabetically',
+  length = 'length',
+}
+
+const getPreperedGoods = (
```

Comment:
There is a typo in the function name `getPreperedGoods`. It should be `getPreparedGoods` to maintain consistency and readability.